### PR TITLE
[PLATFORM-328] Update streamr-client to 1.0.0

### PR DESCRIPTION
**Pull request by timoxley**
Updates streamr-client to 1.0.0.

### CODE REVIEW

File: app/src/userpages/components/WebComponents/StreamrWidget/index.jsx [hunk 1]
```diff
@@ -96,7 +96,7 @@ export default class StreamrWidget extends Component<Props> {
             if (this.stream && !this.subscription) {
                 this.subscription = this.client.subscribe({
                     stream: this.stream,
-                    authKey: subscriptionOptions.authKey,
+                    apiKey: subscriptionOptions.apiKey,
```
[the remark this answers is not in the document]
Comment (timoxley) [reply app/src/userpages/components/WebComponents/StreamrWidget/index.jsx]: Correct, these can probably be removed.

File: app/src/editor/shared/components/Client.jsx [hunk 2]
```diff
@@ -54,7 +57,7 @@ class ClientProviderComponent extends Component {
 
     send = async (rest) => (
         services.send({
-            authKey: this.props.authKey,
+            apiKey: this.props.apiKey,
```
[the remark this answers is not in the document]
Comment (timoxley) [reply app/src/editor/shared/components/Client.jsx]: Good point

[the remark this answers is not in the document]
Comment (timoxley) [reply app/src/editor/shared/components/Client.jsx]: oh, hang on, no this is `services.send`, an HTTP call, doesn't use the client at all: https://github.com/streamr-dev/streamr-platform/blob/development/app/src/editor/shared/services.js#L18-L34

[the remark this answers is not in the document]
Comment (juhah) [reply app/src/editor/shared/components/Client.jsx]: Yep, you're right!

**Pull request merged**